model/riscv_insts_fext.sail

```diff
@@ -389,6 +418,13 @@ function process_fload32(rd, addr, value) =
     MemException(e)  => { handle_mem_exception(addr, e); RETIRE_FAIL }
   }
 
+val process_fload16 : (regidx, xlenbits, MemoryOpResult(bits(16)))
```

Comment:
Really the functions should be the other way round, but that's not your problem, so this is the right thing to do in the situation